### ISSUE TITLE
create cart sign in page

### ISSUE THREAD

**Issue by Anish005** [su-brat/cart-saas-frontend#12]
user using the cart for the first time or the sign up sign in pages....necessary features to be added accordingly , gradually  , for now basically performing authentication with via  backend

**Comment by su-brat** [comment]
- Are we implementing sign in or sign up page? (Since earlier we discussed to not have sign in and sign up from our side and just get the user details from the merchant side)
- If we decide to implement, lets design it first. Also I feel this can go to last of priorities, after covering all the basic requirements like cart page, address section etc.

**Comment by Anish005** [comment]
yeah we won't have it keeping it as a functionality of testing certain features before dealing directly with the merchant or could modify it to the  home page like introduction,mentioning about cart usages just like other service providers have an intro about there product

**Comment by Anish005** [comment]
yeah least of priority but wanted to implement  certain authentication while learning and establish checks for other project usages  not be merged to the dev until other works are done

**Comment by su-brat** [comment]
💭 let's discuss on call whenever we setup next..
